client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -484,3 +484,10 @@
 		display: none;
 	}
 }
+
+// Customizations to the payments modal when it shows on Launchpad
+.memberships__dialog-sections-type,
+.memberships__dialog-sections-message,
+.memberships__dialog-advanced-options {
+	display: none;
+}
```

Comment:
What would happen if we simply didn't render those elements?

Wrap them with `{ ! isNewsletterPlan && <FormFieldset /> }` or similar?

Reply:
If we add that logic (with ! isNewsletterPlan) to the component itself, it sounds like we're assuming that a person with a newsletter site would never want to add different non-newsletter products or update thank you message or edit the advanced options, even from the Earn page. Seems like that could be quite limiting?

What I'd been thinking here is we're showing the simplified version for onboarding, but letting them go access/edit all the options from Earn if they want later.

If we just wanted implement this via component logic rather than CSS-only, we could also do something like { ! isLaunchpad && <FormFieldset /> }. 

But for my part, I'd still probably prefer to show all the options on the earn page, even for newsletter users. Not a strong view though - if we prefer otherwise, I'm happy to do that.

Reply:
Yeah, `isLaunchpad` or something similar works but main point was that we'd not even render the components in the launchpad version of the modal.

It's fine to keep showing them in the Earn panel for now.